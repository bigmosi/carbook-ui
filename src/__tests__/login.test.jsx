import React from 'react';
import '@testing-library/jest-dom';
import renderWithProviders from '../utils/test-utils';
import LoginForm from '../features/auth/components/LoginFrom';

describe('Login component', () => {
  const { asFragment } = renderWithProviders(<LoginForm />);
  const firstRender = asFragment();

  it('should render correctly', () => {
    expect(firstRender).toMatchSnapshot();
  });
});
