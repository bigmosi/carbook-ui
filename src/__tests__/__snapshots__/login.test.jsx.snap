// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <form
      class="form"
    >
      <div>
        <label
          class="label"
          for="username"
        >
          Username
          <input
            class="input"
            id="username"
          />
        </label>
      </div>
      <div>
        <label
          class="label"
          for="password"
        >
          Password
          <input
            class="input"
            id="password"
            type="password"
          />
        </label>
      </div>
      <div>
        <button
          class="button"
          type="button"
        >
          Log in
        </button>
      </div>
    </form>
    <small
      class="auth-footer"
    >
      <span>
        Dont have an account? 
      </span>
      <a
        href="/register"
      >
        Regiter
      </a>
    </small>
  </div>
</DocumentFragment>
`;
